lib/api/src/grpc/proto/qdrant.proto

```diff
@@ -7,6 +7,7 @@ import "points_service.proto";
 import "points_internal_service.proto";
 import "qdrant_internal_service.proto";
 import "raft_service.proto";
+import "shard_snapshots_service.proto";
```

Comment:
I think that by convention, we better call it `snapshots_internal_service.proto`.

Reply:
@ffuugoo Before I hit approve, what do you think about this one?

Reply:
I don't have a strong opinion. I think both are fine. There's not that much in `shard_snapshots_service.proto` that is inherently "internal" (only `ShardSnapshotPriority` having `ShardTransfer` variant).